apps/playground/src/app/modules/components/password-module/password-module.component.ts

```diff
@@ -31,43 +31,44 @@ export class PasswordModuleComponent implements OnInit, OnDestroy {
 
   private unsubscribe$ = new Subject<void>();
 
-  constructor(private formBuilder: NonNullableFormBuilder) { }
+  constructor (private formBuilder: NonNullableFormBuilder) {
+  }
 
-  ngOnInit(): void {
+  ngOnInit (): void {
     this.initForms();
   }
 
-  private initForms(): void {
+  private initForms (): void {
```

Comment:
I am more on the "function parentheses right after the function's name" team so I'll give you on Mattermost the file that describes my settings for Typescript files in Webstorm and you'll be able to work with it